bin/tictactoe

```diff
@@ -1,3 +1,17 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+=begin
+Greet the user with a message.
+Prompt the user for what kind of game they want to play, 0,1, or 2 player.
+Ask the user for who should go first and be "X".
+Use the input to correctly initialize a Game with the appropriate player types and token values.
+When the game is over, the CLI should prompt the user if they would like to play again and allow them to choose a new configuration for the game as described above. If the user doesn't want to play again, exit the program.
+=end
+
+puts "Greetings! Word on the street has it that you're pretty good at Tic Tac Toe. Let's see what you got!"
+
+
+Game.start
```

Comment:
Great bin file, short length is ideal. However, we would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.